From ecdc6753c0b6e8bb7802a8631cd0346b70d66695 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Mon, 5 Jun 2017 12:35:30 +0800
Subject: [PATCH 1844/5242] MLK-15005-02 arm64: imx8qm: add lvds0/lvds1/hdmi
 LIS ipg clock

commit  aa0ca92c3b96b3841c16ef8fae631ccc7327162c from
https://source.codeaurora.org/external/imx/linux-imx.git

Add lvds0/lvds1/hdmi LIS ipg clock.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |   12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 079ec39..c129734 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -843,7 +843,13 @@
 		interrupt-controller;
 		interrupt-parent = <&gic>;
 		#interrupt-cells = <2>;
-		power-domains = <&pd_hdmi_i2c0>;
+		clocks = <&clk IMX8QM_HDMI_DIG_PLL_CLK>,
+				<&clk IMX8QM_HDMI_LIS_IPG_CLK>;
+		clock-names = "pll", "ipg";
+		assigned-clocks =  <&clk IMX8QM_HDMI_DIG_PLL_CLK>,
+				<&clk IMX8QM_HDMI_LIS_IPG_CLK>;
+		assigned-clock-rates = <675000000>, <84000000>;
+		power-domains = <&pd_hdmi>;
 	};
 
 	i2c0_hdmi: i2c@56266000 {
@@ -867,6 +873,8 @@
 		interrupt-controller;
 		interrupt-parent = <&gic>;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_LVDS0_LIS_IPG_CLK>;
+		clock-names = "ipg";
 		power-domains = <&pd_lvds0>;
 	};
 
@@ -891,6 +899,8 @@
 		interrupt-controller;
 		interrupt-parent = <&gic>;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_LVDS1_LIS_IPG_CLK>;
+		clock-names = "ipg";
 		power-domains = <&pd_lvds1>;
 	};
 
-- 
1.7.9.5

